src/OnlineSales/Infrastructure/SampleEmailSenderTask.cs

```diff
@@ -0,0 +1,70 @@
+﻿// <copyright file="SampleEmailSenderTask.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.IO;
+using System.Net.Http.Headers;
+using Nest;
+using OnlineSales.DTOs;
+using OnlineSales.Entities;
+using OnlineSales.Interfaces;
+
+namespace OnlineSales.Infrastructure;
+
+public class SampleEmailSenderTask : ITask
```

Comment:
This class looks like something that should be part of committed code because it references some test files and has hardcoded emails. Please remove.